### PR TITLE
Move ELB lightweight module config into integration

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?
This PR is to move lightweight module configuration from metricbeat into integrations. Also this PR adds collection to `HealthyHostCount` and `UnHealthyHostCount` for ApplicationELB. 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by tommyers-elastic** [comment]
this LGTM - but i don't really know what i'm looking at yet 😜  - would you mind walking me through the changes @kaiyan-sheng ?

also - whats the story with testing these kinds of changes? i've seen a couple or PRs already which appear to be copy-pasted configurations - is there any way to guarantee there are no typos or copy-paste errors?

thanks!

**Comment by kaiyan-sheng** [comment]
@tommyers-elastic Yes definitely! When we first came up with [lightweight modules](https://www.elastic.co/blog/introducing-metricbeat-light-modules), we have the configuration files in Beats, such as https://github.com/elastic/beats/blob/main/x-pack/metricbeat/module/aws/elb/manifest.yml. When we migrated all the metricsets into the AWS package as individual integrations, the migration script we use migrated the light module directly. This means the configuration for collecting ELB metrics is still in the Beats repo [here](https://github.com/elastic/beats/blob/main/x-pack/metricbeat/module/aws/elb/manifest.yml) instead of in the Integrations repo. When we find a bug, for example, this bug in ELB with missing `HealthyHostCount` and `UnHealthyHostCount` for ApplicationELB, in order to fix it, we have to make the change in Beats repo instead of directly here. Fixing it in Beats repo means we have to wait for the next minor release in order for users to pick up the bug fix, which might take a long time. That's the reason why we are doing the migration to move lightweight module configs into the Integrations repo.

For testing: great question! This is not very reliable right now. What we have is a check `static test: Verify sample_event.json – aws.elb_metrics` in CI that checks if the fields in `sample_event.json` event are all documented under fields folder. If not CI would fail. The reason why I say it's not very reliable is because the `sample_event.json` is generated by the PR contributor manually and most of the time, the event documented in `sample_event.json` only includes some of the fields, not all.

File: packages/aws/data_stream/elb_metrics/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -32,4 +32,103 @@ tags_filter: {{tags_filter}}
...
+  - ProcessedBytes
+  - RejectedConnectionCount
+  - RequestCount
+  - RuleEvaluations
+- namespace: AWS/ApplicationELB
+  resource_type: elasticloadbalancing
+  statistic: ["Average"]
+  name:
+  - ConsumedLCUs
+- namespace: AWS/ApplicationELB
```
Comment (aspacca) [inline packages/aws/data_stream/elb_metrics/agent/stream/stream.yml.hbs:94]: 👍

File: packages/aws/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 format_version: 1.0.0
 name: aws
 title: AWS
-version: 1.16.3
+version: 1.16.4
```
Comment (aspacca) [inline packages/aws/manifest.yml:4]: this version was already used by https://github.com/elastic/integrations/pull/3499/files#diff-6ea379ea4e907be5cc25a65c404d5bb98841ab15439b2c43836118150c90c9afR4

**Review by aspacca — COMMENTED** [review]
please, @kaiyan-sheng , see my comment about the integration version

File: packages/aws/manifest.yml (hunk 2 above)
Comment (kaiyan-sheng) [reply packages/aws/manifest.yml:4]: haha yep I will resolve the merge conflict :) Thanks!

File: packages/aws/changelog.yml [hunk 3]
```diff
@@ -1,9 +1,17 @@
 # newer versions go on top
-- version: "1.16.4"
+- version: "1.16.5"
   changes:
     - description: Move ELB metrics config from beats to integrations
       type: enhancement
       link: https://github.com/elastic/integrations/pull/3474
+- version: "1.16.4"
```
Comment (aspacca) [inline packages/aws/changelog.yml:7]: that's strange: lines 7-14 should not be part of the diff

**Review by aspacca — APPROVED** [review]

**Review by aspacca — APPROVED** [review]

**Pull request merged**